tests/suite/test_annotations.py

```diff
@@ -566,7 +566,7 @@ def test_standard_ingress(
         unexpected_strings,
     ):
         initial_events = get_events(kube_apis.v1, annotations_setup.namespace)
-        initial_count = get_event_count(annotations_setup.ingress_event_text, initial_events)
+        get_event_count(annotations_setup.ingress_event_text, initial_events)
```

Comment:
This whole line can be removed?

Reply:
good catch.  And the one above as initial events is not used elsewhere. I am assuming these get functions have no side effects, but we will know when the tests run